secure_fw/CMakeLists.txt

```diff
@@ -47,10 +47,18 @@ target_compile_options(tfm_s
         ${COMPILER_CP_FLAG}
 )
 
+if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+    # To get debug symbols of function called through veneer functions
```

Comment:
Let us give a detailed description of what the problem is and how we are solving it here.

It should include at least everything relevant from the commit message.

It should also include a description of why this does not cause bloat.

And be honest about that we are uncertain of the root cause of the issue and that we suspect a bug in LD.